services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_api.py

```diff
@@ -239,16 +239,15 @@ def _make_pending() -> Tuple[ServiceState, str]:
         # it is looking for a service or something with no error message
         return _make_pending()
 
-    task_status = last_task["Status"]
-    service_state, message = extract_task_state(task_status=task_status)
+    service_state, message = extract_task_state(task_status=last_task["Status"])
 
     # to avoid creating confusion for the user, always return the status
     # as pending while the dynamic-sidecar is starting, with
     # FAILED and COMPLETED and RUNNING being the only exceptions
     if service_state not in NO_PENDING_OVERWRITE:
         return ServiceState.PENDING, message
 
-    return extract_task_state(task_status=task_status)
+    return service_state, message
```

Comment:
so then the backend required no changes? just the front-end?

Reply:
That would be my idea. If there is an error while in pending state, it will be logged by the fronted.